.github/workflows/2_auto_publish_release.yml

```diff
@@ -29,8 +29,7 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v5
       with:
-        # return to 3.x once cylc-flow is compatible with 3.10+ (pyuv)
-        python-version: '3.9'
+        python-version: '3.x'
```

Comment:
I thought we needed to specify the `setuptools` version in `pyproject.toml[build-system]` but it turns out it was just this after all